core/storage/user/gae_models.py

```diff
@@ -204,7 +203,7 @@ def export_data(user_id):
         }
 
     @classmethod
-    def get_new_id(cls, unused_entity_name):
+    def get_new_id(cls, unused_entity_name=''):
```

Comment:
Why this change?

Reply:
Mistake, fixed.

Reply:
Not fixed, AFAICT. Please proofread your code changes.

Reply:
Sorry, done.